src/os/ui/file/kml/abstractkmlexporter.js

```diff
@@ -606,6 +607,8 @@ class AbstractKMLExporter extends ZipExporter {
     this.addGeometryNode(item, element);
 
     var fields = this.getFields(item);
+    fields.push(RecordField.RING_OPTIONS);
```

Comment:
Can this be included in the `getFields` result? It feels like a random one-off to add in here.

For the standard Export path, we have never exported internal fields like this and only include them if you choose Export Places, which is intended to preserve all OpenSphere-specific metadata. The normal Export path should export geometries/styles as currently rendered on the map, and no additional metadata because it's intended to be consumed by external apps/services like Google Earth.